app/assets/javascripts/reports/reports_datatable.js

```diff
@@ -254,14 +253,15 @@
     updateButtons();
   }
 
-  function generateReportRequest(pathAttrName) {
-    if (CHECKED_REPORTS.length === 1) {
-      let row = $(".report-row[data-id='" + CHECKED_REPORTS[0] + "']");
+  function generateReportRequest(pathAttrName, id) {
+    if (CHECKED_REPORTS.length === 1 || id) {
+      let row = $(".report-row[data-id='" + (id || CHECKED_REPORTS[0]) + "']");
       animateSpinner();
       $.post(row.data(pathAttrName), function(response) {
         animateSpinner(null, false);
         HelperModule.flashAlertMsg(response.message, 'success');
         checkProcessingStatus(row.data('id'));
```

Comment:
Maybe it will be more efficient to call checkProcessingStatus without delay instead of full table reload?

Reply:
The issue was in the missed class name on Docx generate button.